app/views/home/index.html.erb

```diff
@@ -1,5 +1,3 @@
 <h1>Hello World!</h1>
-<% if flash[:success] %>
-  <div class="success"><%= flash[:success] %></div>
-<% end %>
 #Add room for errors here. 
+
```

Comment:
✂️ 

This is a whitespace error. And the comment above shouldn't be there either. (Pretty sure it would output on this page since it's a Ruby-style comment and not an HTML comment. )